nomos-services/cryptarchia-consensus/src/lib.rs

```diff
@@ -633,3 +797,28 @@ async fn mark_in_block<Payload, Item, Key>(
         .await
         .unwrap_or_else(|(e, _)| tracing::error!("Could not mark items in block: {e}"))
 }
+
+async fn mark_blob_in_block<BlobId: Debug>(
+    sampling_relay: SamplingRelay<BlobId>,
+    blobs_id: Vec<BlobId>,
+) {
+    if let Err((_e, DaSamplingServiceMsg::MarkInBlock { blobs_id })) = sampling_relay
+        .send(DaSamplingServiceMsg::MarkInBlock { blobs_id })
+        .await
+    {
+        error!("Error marking in block for blobs ids: {blobs_id:?}");
```

Comment:
I am actually seeing a couple of places where errors are just logged. 
What would actually the consequence be of an error here, for example? Is logging enough?

Reply:
This should never happen as it is internally to the node (msg from service to service). We could actually panic here. Or try to recover. There is no clear policy on this cases. Ideas and suggestions are welcome (open an issue for them).

Reply:
I have learnt in other contexts that when a node encounters an issue it can't actually recover from, it should shutdown, but not panic.

The question here is, what does such an error as this one mean here. Is it fatal, an unrecoverable situation? Actually not really, but something is very fishy still.

Still, a node shutdown should be preferred to a panic IMO.

Reply:
At the moment there is no really a difference between shutdown and panic. 
* A panic should be used when a path is a **bug**, meaning that the programmed logic is wrong, it should be fixed. 
* An error that does not matter (the node may continue) should be logged and execution should continue.
* When something neither the node nor we expect  then as you said it should be shutdown/restart.

In this specific case it was logged so we can see something went wrong then we should try to recover the relay later on (as it was broken), and if its not possible then the node could shut down. But still do not have any recoverable measures in place (anywhere). Not something that we are prioritising now. 
But if you are interested it would be really beneficial to prepare a design for such policies!